<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Avaliação</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>-190</x>
        <y>0</y>
        <width>1081</width>
        <height>651</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Pictures/Studio Ghibli - Anime Wallpaper 01.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Animação :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboFilmes">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A Viagem de Chihiro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Meu Amigo Totoro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Serviço De Entregas Da Kiki</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vidas Ao Vento</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>O Conto Da Princesa Kaguya</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Castelo Animado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Túmulo De Vagalumes </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>As Memórias de Marnie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Reino Dos Gatos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ponyo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Princesa Mononoke</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sussuros Do Coração</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tartaruga Vermelha</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>40</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Studio Ghibli - Filmes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Nota :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);

</string>
      </property>
      <property name="text">
       <string> *(em minutos)</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabelaFilmes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>601</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);
background-color: rgb(85, 170, 255);</string>
      </property>
      <column>
       <property name="text">
        <string>Animação</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Duração</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gênero</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Assistido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Avaliador</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>153</width>
        <height>19</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Já assistiu?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Sim</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Não</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineNota">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboGender">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Aventura</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ação</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fantasia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Romance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Drama</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Guerra</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Aniparô</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sci-Fi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Esportes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Isekai</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>125</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Gênero :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineTempo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Duração :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>56</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Avaliador :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineAvaliador">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>200</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnApagar">
         <property name="text">
          <string>Excluir</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnInserir_2">
         <property name="text">
          <string>Inserir Avaliação</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>70</y>
        <width>154</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Ordenar Por :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboOrdenar">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Animação</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gênero</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>-10</y>
        <width>991</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Pictures/Screenshots/Captura de Tela (114).png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Math TeX Gyre</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
        <strikeout>true</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Dados Gerais</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>265</width>
        <height>83</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>DejaVu Math TeX Gyre</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Minutos Assistidos :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDura">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCarregar"/>
    <addaction name="actionSalvar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
